actors/builtin/power/power_actor.go

```diff
@@ -494,10 +491,8 @@ func (a Actor) processDeferredCronEvents(rt Runtime) {
 					continue
 				}
 
-				// Starting version 7, decrement miner count to keep stats consistent.
-				if rt.NetworkVersion() >= network.Version7 {
-					st.MinerCount--
-				}
+				// Decrement miner count to keep stats consistent.
+				st.MinerCount--
```

Comment:
Did we ever fix the count in an upgrade?

Reply:
Yes